net/apfree-wifidog/files/wifidogx.conf

```diff
@@ -1,13 +1,9 @@
-config wifidog
-	option	gateway_interface	'br-lan'
-	option	auth_server_hostname	'change wifidog.kunteng.org.cn to your auth server domain or ip'
-	option	auth_server_port	8001
-	option	auth_server_path	'/wifidog/'
-	option	check_interval		60
-	option	client_timeout		5
-	option	httpd_max_conn		200
-	option	pool_mode			1
-	option	thread_number		5
-	option	queue_size			20
-	option	wired_passed		0
-	option	enable		0
+config wifidogx
```

Comment:
Please be aware that this section type name change will break compatibility and users' old config will be rendered useless.  OpenWrt tries hard to not break users' existing config even in master branch.

Reply:
@yousong yes I agree with u, there are 2 reason why I change it
1, apfree wifidog was received by openwrt a few days ago, there are no history problems
2, mixed use wifidog and wifidogx will confuse users
that's why I changed it

Reply:
restore it

Reply:
It's not that such changes are forbidden, just that we need to be more careful when making changes that can cause behavior changes for existing users.  If you really intend to make such changes, it's also okay, but be sure to present the reasons to back it.

I noticed that are also uci options dropped in this pull request.  These also deserve a note in the commit message.  Thank you.

Reply:
@yousong as u see, this version has replaced libhttpd with libevent, there were very big change. for new version thread pool mode was deprecated